quarkus/tests/integration/src/test/resources/org/keycloak/it/cli/approvals/cli/help/HelpCommandTest.testStartDevHelpAll.approved.txt

```diff
@@ -144,10 +144,21 @@ Vault:
 
 Logging:
 
---log-format <format>
-                     The format of log entries. If the format has spaces in it, you need to escape
-                       the value such as "<format>". Default: %d{yyyy-MM-dd HH:mm:ss,SSS} %-5p [%c]
-                       (%t) %s%e%n.
+--log <handler>      Enable one or more log handlers in a comma-separated list. Available log
+                       handlers are: console,file Default: console.
+--log-console-color <true|false>
+                     Enable or disable colors when logging to console. Default: true.
+--log-console-format <format>
+                     The format of plain console log entries. If the format has spaces in it,
+                       escape the value using "<format>". Default: %d{yyyy-MM-dd HH:mm:ss,SSS} %-5p
+                       [%c] (%t) %s%e%n.
+--log-console-output <plain|json>
```

Comment:
I keep thinking if `plain` is the correct value for the `default` logging mechanism XD

Reply:
Hum, you are right. I think we should rename to default here, to stay consistent with our internal SPI/Factory approach. Will do :)

Reply:
Done